interface/src/components/homepage.js

```diff
@@ -16,8 +16,8 @@ class homepage extends Component{
                 <div className="steps">
                     <div><span>Steps</span></div>
                     <ol>
-                        <li>Fill out Metadata in this google spreadsheet template (How to do this).</li>
-                        <li>Prepare a shapefile or table with plot polygons (How to do this).</li>
+                        <li>Fill out Metadata in the google spreadsheet template (<a href="https://docs.google.com/spreadsheets/d/1c_5j7q3TO6gQ24KSopE-_LXA5HhfsUEqMupckGZAbo8/edit#gid=0" rel="noopener noreferrer" target="_blank" style= { { color: "blue"} }>How to do this</a>).</li>
```

Comment:
The color should be what the user prefers - could impact Blue color blind people

Reply:
isn't this just for a hyperlink? From quick research, it looks like blue hyperlinks can be distinguished from black text. e.g. [this site](https://gomakethings.com/better-link-accessibility-for-the-color-blind) says "The blue links on this site are easily distinguishable from plain text to all three forms of color blindness."

For the plot colors, you can find color-blind friendly palettes from here: https://colorbrewer2.org/#type=diverging&scheme=PuOr&n=5